webpack/components/extensions/HostDetails/Tabs/RemoteExecutionConstants.js

```diff
@@ -9,5 +9,6 @@ export const REX_FEATURES = {
   KATELLO_HOST_TRACER_RESOLVE: 'katello_host_tracer_resolve',
   KATELLO_HOST_ERRATA_INSTALL_BY_SEARCH: 'katello_errata_install_by_search',
   KATELLO_HOST_MODULE_STREAM_ACTION: 'katello_module_stream_action',
+  KATELLO_HOST_CHANGE_CONTENT_SOURCE: 'katello_change_content_source',
```

Comment:
This will be unused in the JavaScript so it's not needed.

Reply:
Thanks for confirmation (I just did when I compared how other features were implemented). Just to understand the others constants are needed because the ui provides run action X and customise REX. Since the ui for change content source already open the REX page for customisation: it won't be needed.

I will remove it.

Reply:
Yeah, we use the other constants in JavaScript when we're composing our API requests to call the REX API directly from the frontend (`/job_invocations/new`), for example [here in customizedRexUrlHelpers](https://github.com/Katello/katello/blob/0250c25c0ca44fff077983aea3541e989bd047c0/webpack/components/extensions/HostDetails/Tabs/customizedRexUrlHelpers.js#L52-L53). Whereas for the change content source feature [here](https://github.com/Katello/katello/blob/0250c25c0ca44fff077983aea3541e989bd047c0/webpack/scenes/Hosts/ChangeContentSource/actions.js#L24-L25), we're actually calling a different API endpoint `/api/v2/hosts/bulk/change_content_source`, and that endpoint doesn't require specifying the REX feature name.